server/api/applicant.js

```diff
@@ -208,20 +208,23 @@ function applicant({ app, db, pgp }) {
       const userId = res.locals.userid;
       const term = req.params.term;
       const dbQuery = `
-SELECT course.id as code, code as codename, name, description, grade, interest, qualification, campuses.campus
-FROM 
-(SELECT * FROM course 
-    WHERE course.id IN (SELECT course FROM section WHERE term=$2)) AS course
-JOIN 
-(SELECT course, campus FROM section 
-    WHERE term=$2) AS campuses
-ON course.id=campuses.course
-LEFT JOIN 
-(SELECT * FROM application 
-    WHERE applicant IN (SELECT id FROM users WHERE username=$1) AND term=$2) AS application
-ON application.course = course.id AND application.campus=campuses.campus
-ORDER BY course.code
-    `;
+SELECT DISTINCT
+course.id as code, 
+code as codename, 
+name, 
+description, 
+grade, 
+interest, 
+qualification, 
+section.campus
+FROM course JOIN section ON course.id=section.course 
+LEFT JOIN (
+    SELECT * FROM application WHERE 
+    applicant IN (SELECT id FROM users WHERE username=$1) AND 
+    term=$2
+) AS application
+ON application.course = course.id AND application.campus=section.campus
```

Comment:
Please test to double-check this is correct.
```suggestion
LEFT JOIN application ON (
    application.applicant IN (SELECT id FROM users WHERE username=$1) AND 
    application.term=section.term AND
    application.course = course.id AND
    application.campus=section.campus)
```